### PR TITLE
Replace unsigned int and update comments for polyline methods

### PR DESCRIPTION
**Pull request by dnesbitt61**

### CODE REVIEW

**Comment by gknisely** [comment]
:ship:

File: src/midgard/pointll.cc [hunk 1]
```diff
@@ -66,7 +66,7 @@ float PointLL::Distance(const PointLL& ll2) const {
 
 float PointLL::Length(const std::vector<PointLL>& pts) {
   float length = 0.f;
-  for (unsigned int i = 0; i < pts.size() - 1; ++i)
+  for (uint32_t i = 0; i < pts.size() - 1; ++i)
```
Comment (kevinkreiser) [inline src/midgard/pointll.cc]: Should be size_t

**Comment by kevinkreiser** [comment]
uint32_t smashes size_t on 64bit systems you should use size_t. Also if you aren't needing to keep the number of bits consistent across systems there is not much point to being explicit in number of bits

**Comment by dnesbitt61** [comment]
To avoid this discussion, how about:  
for (auto pt1 = pts.begin(), pt2 = pt1 + 1; pt2 < pts.end(); pt1++, pt2++) {
    length += pt1->Distance(*pt2);
  }

**Pull request merged**